six/two/two.cpp

```diff
@@ -269,11 +269,16 @@ bool Two::getCheck(SixPyObject *py_class, const char *init_config_str, const cha
     }
 
     instances = PyTuple_New(1);
+    if (instances == NULL) {
+        setError("could not create tuple for instances: " + _fetchPythonError());
+        Py_XDECREF(instance); // we still own the reference to instance, so we need to decref it here
```

Comment:
I think we might want to decref `instance` before going to `done` in line 268. It's surely a rare scenario, but in that case `instance != NULL` so we hold a reference to something.